yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -445,9 +467,16 @@ func checkAndHandleSnapshotComplete(config *dbzm.Config, status *dbzm.ExportStat
 				utils.PrintAndLog("Waiting to initialize export of change data from target DB...")
 				logFilePath := filepath.Join(exportDir, "logs", fmt.Sprintf("debezium-%s.log", exporterRole))
 
-				err := utils.WaitForLineInLogFile(logFilePath, "Beginning to poll the changes from the server", 3*time.Minute)
-				if err != nil {
-					return false, fmt.Errorf("failed to poll for message in log file: %w", err)
+				// In case of the logical replication connector, we don't need to wait for the log message.
+				// The momemnt a replication slot is created, we are guaranteed that we will receive events.
+				// In the case of the old connector, there is no such explicit operation to 'start' CDC.
+				// It used to happen implicitly, which is why we have to wait for a log message.
+				if !msr.UseLogicalReplicationYBConnector {
+					pollingMessage := "Beginning to poll the changes from the server"
```

Comment:
The printAndLog and logFilePath (lines 467 and 468) should also be inside this if block